test/celeritas/optical/Cerenkov.test.cc

```diff
@@ -378,6 +392,53 @@ TEST_F(CerenkovTest, TEST_IF_CELERITAS_DOUBLE(generator))
         EXPECT_SOFT_EQ(401.5, avg_engine_samples);
     }
 }
+
+//---------------------------------------------------------------------------//
+
+TEST_F(CerenkovTest, TEST_IF_CELERITAS_DOUBLE(pre_generator))
```

Comment:
Since the generator test above relies on the pregenerator working correctly, I'd move this test above the `generator` test.